exec/java-exec/src/test/java/org/apache/drill/TestAltSortQueries.java

```diff
@@ -64,9 +73,9 @@ public void testJoinWithLimit() throws Exception{
         "  nations.N_NAME,\n" +
         "  regions.R_NAME\n" +
         "FROM\n" +
-        "  dfs_test.`[WORKING_PATH]/../../sample-data/nation.parquet` nations\n" +
+        "  dfs.`/sample-data/nation.parquet` nations\n" +
```

Comment:
I accidentally changed it to dfs, will switch it back to dfs_test for consistency. <b>dfs</b> and <b>dfs_test>/b> are configured to point to the same temp directories so they are in fact interchangeable. This configuration happens in the openClient method of BaseTestQuery and in the configureStoragePlugins method of the ClusterFixture. 

Throughout making this change I was scratching my head as to why <b>dfs_test</b> is used instead of <b>dfs<b>. After going through the code I don't see any reason as to why dfs_test was created in the first place. I think we should actually remove <b>dfs_test</b> and just use <b>dfs</b> everywhere for uniformity. I'll go ahead and do that now.

Reply:
Just mentioned this above but will repeat here.I have now removed **dfs_test** completely. There was no reason for it to be added and it was inconsistently being mixed with **dfs**. If you want to query a file on the local filesystem that is not on the classpath just using **dfs** will be sufficient now.